File: app/src/main/java/theo/base/ui/main/MainActivity.java
package theo.base.ui.main;

import android.os.Bundle;

import theo.base.R;
import theo.base.ui.base.BaseActivity;

public class MainActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        setFragment(new MainFragment(), true);
    }
}

File: app/src/main/kotlin/theo/base/ui/main/MainActivity.kt
package theo.base.ui.main

import android.os.Bundle

import theo.base.R
import theo.base.ui.base.BaseActivity

class MainActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setFragment(MainFragment(), true)
    }
}
